src/web/crate_details.rs

```diff
@@ -180,11 +183,11 @@ impl CrateDetails {
             description: rows.get(0).get(4),
             rustdoc_status: rows.get(0).get(11),
             target_name: rows.get(0).get(16),
-            default_target: rows.get(0).get(25),
+            default_target: rows.get(0).get(26),
```

Comment:
Oh boy, we have some technical debt stacking up. Once this PR is in I think we should start think about getting things by name instead of by index: https://docs.rs/postgres/0.15.2/postgres/rows/struct.Row.html#method.get